README.md

```diff
@@ -1,24 +1,178 @@
 [![Build Status](https://travis-ci.org/HumanCellAtlas/metadata-api.svg?branch=develop)](https://travis-ci.org/HumanCellAtlas/metadata-api)
 [![Coverage Status](https://coveralls.io/repos/github/HumanCellAtlas/metadata-api/badge.svg?branch=develop)](https://coveralls.io/github/HumanCellAtlas/metadata-api?branch=develop)
 
-## The HumanCellAtlas metadata API
+The HumanCellAtlas metadata API
+===============================
 
-A light-weight wrapper library around the JSON metadata in HCA data bundles.
-The library serves two purposes: providing convenient programmatic access to a
-subset of metadata attributes as well as decoupling clients from schema changes
-that would break direct access to the metadata.
+## Overview
 
+Metadata API serves as a light-weight wrapper library around the JSON metadata 
+in HCA data bundles. This library serves two purposes: providing convenient 
+programmatic access to a subset of metadata attributes as well as decoupling 
+clients from schema changes that would break direct access to the metadata.
+
+## Requirements
+
+- Python 3.6
 
 ## Installation
 
 Version 1.0 will be on PyPI but until then we need to install from GitHub: 
 
 ```
-virtualenv -p python3 foo
-source foo/bin/activate
-pip install "git+git://github.com/HumanCellAtlas/metadata-api@master#egg=hca-metadata-api[dss]"
+$ virtualenv -p python3 foo
+$ source foo/bin/activate
+$ pip install "git+git://github.com/HumanCellAtlas/metadata-api@master#egg=hca-metadata-api[dss]"
 ```
 
 You can omit `[dss]` at the end of the `pip` invocation if you don't need
 the download helper this library provides and don't want to pull in the HCA CLI
 distribution the helper depends on.
+
+## Testing
+
+To run unit tests, run 
+```
+$ make test
+```
+
+
+## Terminology
```

Comment:
Remove the Terminology section. It is inaccurate, incomplete and confuses terminology with class documentation. 

Don't duplicate detailed documentation in the README. Make sure that the docstrings have all the details, simply list what's available but don't give specific details. Example: your section on the helpers duplicate docstring. Documentation is extremely expensive to maintain, if you duplicate details your multiplying the cost of maintenance.

Always put backquotes around class, method and function names. You are inconsistent in the use of backquotes.

Make sure no horizontal scrolling is necessary in any preformatted blocks.